backend/src/routes/doctorOrders.js

```diff
@@ -177,6 +177,14 @@ router.patch('/api/updateRx/:id/pickedUp', async (req, res) => {
       }
     });
     console.log('Sent RxFill to EHR and received status from EHR', status.data);
+
+    const remsAdminStatus = await axios.post('http://rems-administrator:8090/ncpdp', rxFill, {
```

Comment:
The rems-administrator endpoint will not work outside of docker. This should be configurable as an environment variable and we can maybe add the capability to forward these messages through the intermediary in the future.